app/_hub/kong-inc/session/_index.md

```diff
@@ -436,6 +543,30 @@ _not_ a problem during session renewal period as renew happens in `access` phase
 
 ## Changelog
 
+**{{site.base_gateway}} 3.2.x**
+* The plugin has been updated to use version 4.0.0 of the `lua-resty-session` library. This introduced several new features, such as the possibility to specify an `audience` for the session.
+The following configuration parameters were affected:
+
+Added:
+  * `audience`
+  * `remember`
+  * `remember_cookie_name`
+  * `remember_rolling_timeout`
+  * `remember_absolute_timeout`
+  * `absolute_timeout`
+  * `request_headers`
+  * `response_headers`
+
+Renamed:
+  * `cookie_lifetime` to `rolling_timeout`
+  * `cookie_idletime` to `idling_timeout`
+  * `cookie_samesite` to `cookie_same_site`
+  * `cookie_httponly` to `cookie_http_only`
+  * `cookie_discard` to `stale_ttl`
```

Comment:
Question: do we still have backwards compatibility for the renamed parameters, or is this a breaking change, where old configurations will no longer work?

Reply:
we have backwards compatibility and no plans to deprecate the renamed parameters. All parameters are still accepted with their "old" names. Because of this, from a gateway perspective, I believe there is no need to update the developer portal documentation, apart maybe for removing the `cookie_renew` field, which is now ignored. I am going to test this today, thanks for pointing it out!

Reply:
@lena-larionova I confirmed that both kong manager and dev portal's sessions still work with or without providing the `cookie_renew` field. It would be a good idea to remove it from all the 3.2.x docs though, like the ones you linked. Should I just go ahead and remove the field from the pages you have linked or is there something to do to version this change correctly?

Reply:
> Should I just go ahead and remove the field from the pages you have linked or is there something to do to version this change correctly?

Hmm. Good question. I _think_ the answer is simply remove it - from what I can tell, it was only a suggested parameter, and never a required one.